B/bmon/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "bmon"
+version = v"4.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/tgraf/bmon/releases/download/v4.0/bmon-4.0.tar.gz" =>
+    "02fdc312b8ceeb5786b28bf905f54328f414040ff42f45c83007f24b76cc9f7a",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/bmon-*/
+export CFLAGS=-I$prefix/include
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+
```

Comment:
We need to manually install the license files:
```suggestion
install_license LICENSE.*
```